lib/components/app/app-menu-item.tsx

```diff
@@ -68,7 +68,9 @@ export default class AppMenuItem extends Component<Props, State> {
   render(): JSX.Element {
     const { icon, id, onClick, subItems, text, ...otherProps } = this.props
     const { isExpanded } = this.state
-    const Element = otherProps.href ? 'a' : 'button'
+    const hasHref = !!otherProps.href
+    const isAbsolute = otherProps.href?.startsWith('http')
```

Comment:
Looks good! One suggestion, but not needed:

```suggestion
    const isAbsolute = hasHref && otherProps.href?.startsWith('http')
```

Reply:
It is not needed. That condition would be redundant, as `otherProps.href?.startsWith(...)` is null if `otherProps.href` is null, and can't be non-null unless `otherProps.href`is non null. Same for your other suggestion.